azure-pipelines/microbuild.before.yml

```diff
@@ -8,13 +8,13 @@ steps:
 - task: ComponentGovernanceComponentDetection@0
   displayName: 🔍 Component Detection
 
-- task: notice@0
-  displayName: 🛠️ Generate NOTICE file
-  inputs:
-    outputfile: $(System.DefaultWorkingDirectory)/obj/NOTICE
-    outputformat: text
-  condition: and(succeeded(), ne(variables['Build.Reason'], 'PullRequest'))
-  retryCountOnTaskFailure: 3 # fails when the cloud service is overloaded
+# - task: notice@0
```

Comment:
If this is because the task fails too frequently, I have an alternative workaround in Library.Template that just needs to merge in.

Reply:
Yeah, I wasn't able to get a test build out

Reply:
#184 should alleviate that.